src/Plugin/Action/DeleteMessage.php

```diff
@@ -70,4 +71,12 @@ public function execute($object = NULL) {
     $this->executeMultiple(array($object));
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function access($object, AccountInterface $account = NULL, $return_as_object = FALSE) {
```

Comment:
Yeah sure i can do that. But the parameter is documented as $object in ActionInterface. Because the interface is very abstract and unspecific. All action plugins in core keep the $object as parameter (SaveComment, AssignOwnerNode, SaveNode).